tests/commands/install_uninstall_test.py

```diff
@@ -725,6 +726,53 @@ def test_commit_msg_legacy(commit_msg_repo, tempdir_factory, store):
     assert second_line.startswith('Must have "Signed off by:"...')
 
 
+def test_post_checkout_integration_failing(tempdir_factory, store):
```

Comment:
hmmm I'm pretty sure the exit code of `post-checkout` is ignored right? so we can probably skip this test 

> This hook cannot affect the outcome of git switch or git checkout.

Reply:
The exit code is ignored by Git but it is still reported by `pre-commit` to the user. I can remove the test but I think it's still a useful thing to have because the hook can actually return non-zero.

Reply:
I'd probably remove it, it doesn't cause a change in behaviour for the git command that's run so the test doesn't really demonstrate anything